Cmfcmf/OpenWeatherMap.php

```diff
@@ -94,7 +102,7 @@ class OpenWeatherMap
      * @throws \Exception If $cache is neither false nor a valid callable extending Cmfcmf\OpenWeatherMap\Util\Cache.
      * @api
      */
-    public function __construct($fetcher = null, $cacheClass = false, $seconds = 600)
+    public function __construct($appid = null, $fetcher = null, $cacheClass = false, $seconds = 600)
```

Comment:
While I like that the `appid` is the first parameter in the constructor, it needs to be adjusted to remain backwards compatibility.